modeltranslation/tests/translation.py

```diff
@@ -193,13 +193,31 @@ class PageTranslationOptions(TranslationOptions):
     fields = ('title',)
 
 
+class SpecificLanguageTranslationOptionsX(TranslationOptions):
+    fields = (
+        'slug',
+    )
+    languages = [x[0] for x in settings.LANGUAGES]
+
+
+class SpecificLanguageTranslationOptionsY(TranslationOptions):
+    fields = (
+        'slug',
+    )
+
+
 # BasePage left unregistered intentionally.
 translator.register(models.Slugged, SluggedTranslationOptions)
 translator.register(models.MetaData, MetaDataTranslationOptions)
 translator.register(models.RichText, RichTextTranslationOptions)
 translator.register(models.Displayable)
 translator.register(models.Page, PageTranslationOptions)
 translator.register(models.RichTextPage)
+translator.register(models.SpecificLanguageModelX, SpecificLanguageTranslationOptionsX)
+translator.register(
+    models.SpecificLanguageModelY, SpecificLanguageTranslationOptionsY,
+    languages=[x[0] for x in settings.LANGUAGES[1]]
```

Comment:
```python
>>> from django.conf import settings
>>> [x[0] for x in settings.LANGUAGES[1]]
['e', 'E']
```

This is not doing what you expect, it's better use hardcoded language codes here.

Reply:
The list of language code has been updated

`languages = ['de', 'en']`